meilisearch/client.py

```diff
@@ -235,6 +254,90 @@ def get_keys(self) -> Dict[str, str]:
         """
         return self.http.get(self.config.paths.keys)
 
+    def create_key(
+        self,
+        options: Optional[Dict[str, Any]] = None
+    ) -> Dict[str, int]:
+        """Creates a new API key.
+
+        Parameters
+        ----------
+        options:
+            Options, the information to use in creating the key (ex: { 'actions': ['*'], 'indexes': ['movies'], 'description': 'Search Key', 'expiresAt': '22-01-01' }).
+            An `actions`, an `indexes` and a `expiresAt` fields are mandatory,`None` should be specified for no expiration date.
+            `actions`: A list of actions permitted for the key. ["*"] for all actions.
+            `indexes`: A list of indexes permitted for the key. ["*"] for all indexes.
+            Note that if an expires_at value is included it should be in UTC time.
+
+        Returns
+        -------
+        keys:
+            The new API key.
+            https://docs.meilisearch.com/reference/api/keys.html#get-keys
+
+        Raises
+        ------
+        MeiliSearchApiError
+            An error containing details about why MeiliSearch can't process your request. MeiliSearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
+        """
+        if options is None:
+            options= {}
+        payload = {**options}
+        return self.http.post(f'{self.config.paths.keys}', payload)
+
+    def udpate_key(
+        self,
+        key: str,
+        options: Optional[Dict[str, Any]] = None
```

Comment:
Each key in the dictionary is optional so I guess technically `options` could be `Optional` also, however if this is the case the update won't do anything.

I'm not sure if it would be better to force at least one key to be present in `options` or to let the user run the update that doesn't do anything.